builder/src/build/patches/investigated/update-webpack-chunks-file/index.ts

```diff
@@ -0,0 +1,32 @@
+import { readdirSync, readFileSync, writeFileSync } from "node:fs";
+
+import { NextjsAppPaths } from "../../../../nextjs-paths";
+import { getUpdatedWebpackChunksFileContent } from "./get-updated-webpack-chunks-file-content";
+
+/**
+ * Fixes the webpack-runtime.js file by removing its webpack dynamic requires.
+ *
+ * This hack is especially bad for two reasons:
+ *  - it requires setting `experimental.serverMinification` to `false` in the app's config file
+ *  - indicates that files inside the output directory still get a hold of files from the outside: `${nextjsAppPaths.standaloneAppServerDir}/webpack-runtime.js`
+ *    so this shows that not everything that's needed to deploy the application is in the output directory...
+ */
+export async function updateWebpackChunksFile(nextjsAppPaths: NextjsAppPaths) {
+  console.log("# updateWebpackChunksFile");
+  const webpackRuntimeFile = `${nextjsAppPaths.standaloneAppServerDir}/webpack-runtime.js`;
+
+  console.log({ webpackRuntimeFile });
```

Comment:
nit: remove?

Reply:
this was already there... so I thought you wanted it 🤷 

I can definitely remove it if it's not wanted 🙂

Reply:
removed 🙂👍